src/_scss/abstract/variables/colors.scss

```diff
@@ -1,10 +1,16 @@
 // Color
+$color-cool-blue:            #205493;
+$color-cool-blue-light:      #4773aa; // lighten($color-cool-blue, 20%)
+$color-cool-blue-lighter:    #8ba6ca; // lighten($color-cool-blue, 60%)
+$color-cool-blue-lightest:   #dce4ef; // lighten($color-cool-blue, 90%)
+
 $color-primary:              #0071bc;
-$color-primary-dark:         #78a0d6;
 $color-primary-darker:       #205493;
 $color-primary-darkest:      #112e51;
-$color-primary-light:        #3f6fb3;
-$color-primary-lighter:      #0071bc;
+$color-primary-dark:         $color-cool-blue;
+$color-primary-light:        $color-cool-blue-light;
+$color-primary-lighter:      $color-cool-blue-lighter;
+$color-primary-lightest:     $color-cool-blue-lightest;
```

Comment:
Updated to be consistent with [USAspending colors](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/_scss/core/_variables.scss#L27), since `$color-primary-light`, `$color-primary-lighter`, and `$color-primary-dark` values didn't quite match up with their names.